stdlib/public/SwiftShims/SwiftStddef.h

```diff
@@ -29,11 +29,19 @@ typedef __SIZE_TYPE__ __swift_size_t;
 #endif
 
 // This selects the signed equivalent of the unsigned type chosen for size_t.
+#if defined(_MSC_VER)
```

Comment:
Hmm, I suppose that this should be:

`#if defined(_MSC_VER) && !defined(__clang__)` ...